<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" />
  <arg name="arm_id" />
  <arg name="load_gripper" default="true" />
  <arg name="x" default="0"   doc="How far forward to place the base of the robot in [m]?" />
  <arg name="y" default="0"   doc="How far leftwards to place the base of the robot in [m]?" />
  <arg name="z" default="0.5" doc="How far upwards to place the base of the robot in [m]?" />

  <group ns="$(arg arm_id)">
    <include file="$(find franka_control)/launch/franka_control.launch" >
      <arg name="robot_ip" value="$(arg robot_ip)" />
      <arg name="load_gripper" value="$(arg load_gripper)" />
    </include>

    <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="cartesian_impedance_example_controller"/>

    <!-- <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" required="false" /> -->
  </group>

</launch>